instance/models/openedx_appserver.py

```diff
@@ -208,6 +208,7 @@ def create_configuration_settings(self):
             # querying the Github API twice), we pass it as a context variable.
             'github_admin_username_list': self.github_admin_username_list,
             'newrelic_license_key': settings.NEWRELIC_LICENSE_KEY,
+            'smtp_relay_settings': self.smtp_relay_settings
```

Comment:
@itsjeyd We are accessing that dict of settings six times in the template. I know it probably has zero practical impact on performance, but constructing the dict of settings six times just to read out one entry still bothers me.

I remembered about django's `{% with %}` tag which I believe was designed for these kind of use-cases (caching variables in templates), so I used it for smtp relay settings, as well as github admin usernames. It makes the vars template more noisy, but I like the fact that it's obvious where the values are coming from just by looking at the template.